src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/FixAbstractMethodsStep.cs

```diff
@@ -289,16 +300,24 @@ void AddNewExceptionMethod (TypeDefinition type, MethodDefinition method)
 
 		public virtual void LogMessage (string message)
 		{
+#if !NET5_LINKER
 			Context.LogMessage (message);
+#else
+			Context.LogMessage (MessageContainer.CreateInfoMessage (message));
```

Comment:
Could we introduce an extension method which provides a `LogMessage(this LinkerContext context, string message)` extension. We could then remove a number of these `#if` blocks in the code.

Reply:
good idea 👍